vumi/persist/txriak_manager.py

```diff
@@ -180,6 +184,16 @@ class TxRiakManager(Manager):
 
     call_decorator = staticmethod(inlineCallbacks)
 
+    def _setup_manager(self):
+        from twisted.internet import reactor
+        self._reactor = reactor
+        self._threadpool = ThreadPool(minthreads=2, maxthreads=2)
```

Comment:
2 threads is a guess, but I don't think it's an unreasonable one. Making it configurable is tricky because it means having different configuration options between the different Riak managers. It's definitely something we should do, but I'd like to get this out first to see if it solves some of our immediate problems.